quickstarts/ansible-automation-controller/config.yml

```
@@ -0,0 +1,43 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor your Ansible Automation Controller?
+  Use New Relic to monitor your Ansible Automation Controller's performance, so that you can get comprehensive and actionable information that your developers can use to successfully troubleshoot problems. Some key capabilities are:
+
+  - Track the efficiency of your Ansible Automation Controller server's business transactions.
+  - Use deep diagnostics to analyze and fix issues with your app's performance.
+  - Detect faults and errors to reduce your app's downtime.
+  - Compare the performance of your Ansible Automation Controller servers.
+  
+  ## Comprehensive monitoring for your Ansible Automation Controller server
+  
+  When you monitor your Ansible Automation Controller server with New Relic, you get all of this full-stack observability out of the box:
+
+  - High value alerts for your Ansible Automation Controller  on key metrics, such as errors, usage metrics, and Awx).
```

```suggestion
  - High value alerts for your Ansible Automation Controller  on key metrics, such as errors, usage metrics, and Awx.
```

Delete stray parenthetical